main.js

```diff
@@ -0,0 +1,16 @@
+var direction = ['north', 'east', 'south', 'west'];
```

Comment:
This aint really doing anything right? 
![image](https://user-images.githubusercontent.com/52118958/98305761-56a2a100-1f77-11eb-8091-940fbd4b5e13.png)